packages/manager/modules/support/src/tickets/new-ticket/issues-form/issues-form.controller.js

```diff
@@ -28,13 +29,17 @@ export default class SupportNewIssuesFormController {
   }
 
   $onInit() {
+    this.defaultServiceName = this.serviceName;
+    this.serviceName = null;
```

Comment:
I think you shouldn't reset a bindings, I don't expect the binding I pass to have a different value inside and outside my component, maybe you should use another variable or add an alias in your bindings declaration to state that your internal name will be `this.defaultServiceName`
```suggestion
    serviceName: '<defaultServiceName'
```

Reply:
Yes good point

Reply:
Done @marie-j